controllers/cloud.redhat.com/utils/utils.go

```diff
@@ -254,22 +254,43 @@ func MakePVC(pvc *core.PersistentVolumeClaim, nn types.NamespacedName, labels ma
 	}
 }
 
-// DeploymentStatusChecker takes a deployment and returns True if it is deemed ready by the logic
-// in the function.
+// DeploymentStatusChecker takes a deployment and returns True if it is deemed
+// ready by the logic in the function.
```

Comment:
Shortening the line length? I thought we used 100?

Reply:
I can change it to 100